src/ExportModule.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { CalculatorTypes } from 'fqm-execution';
+import * as states from './states/states';
+import { factory } from './states/factory';
+
+interface moduleJSONType {
+  name: string;
+  remarks: string[];
+  states: {
+    Initial: states.InitialState;
+    Terminal: states.TerminalState;
+    [key: string]: states.BaseState;
+  };
+}
+
```

Comment:
Might be useful to include a docstring for what the `exportModule` function is achieving